npm/private/npm_translate_lock.bzl

```diff
@@ -400,6 +400,13 @@ def npm_translate_lock(
                 out = "patches.list",
             )
             ```
+            
+            Once you have created this file, you need to create an empty `patches.list` file before generating the first list. You can do this by running
+            ```
+            touch patches/patches.list
+            ```
+            
+            Finally, write the patches file at least once to make sure all patches are listed. This can be done by running `bazel run //patches:patches_update`
```

Comment:
nit: period at the end.

Reply:
```suggestion
            Finally, write the patches file at least once to make sure all patches are listed. This can be done by running `bazel run //patches:patches_update`.
```